### ISSUE TITLE
tint2/virtualbox window bug

### ISSUE THREAD

**Issue by vileator** [void-linux/void-packages#14743]
* xuname:  
  Void 5.1.15_1 x86_64 AuthenticAMD notuptodate rFFF

* package:  
tint2-16.7_1
 virtualbox-6.0.12_1

I've tint2-16.7_1 and virtualbox-6.0.12_1 installed and running, but when the tint2 panel shows/hides from the mouseover event the virtualbox windows get shifted down and right a few pixels(the windows still keep their same WxH dimensions), and doing this may times eventually moves all the virtualbox windows to the bottom right corner of screen. This behavior doesn't affect the window when it's rolled up or minimized though. Attached is my tint2 config.
[tint2rc.txt](https://github.com/void-linux/void-packages/files/3659023/tint2rc.txt)

**Comment by obunden** [comment]
Couldn't import your config, so I just enabled auto hide and it didn't work well with bspwm that I normally use. So i tried fluxbox and I got the same issue that you describe. This should probably be reported upstream to tint2 or to your wm of choice.

A "fix" in the meantime is to use "Match the hidden panel size" in tint2 (It's located under Panel -> Maximized windows)

Another option would be to swap to something wm to something like i3 that works fine with tint2 and can handle the window resize/tiling for you.

Good luck!
